projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -157,6 +191,54 @@ public function require_admin_privilege_callback() {
 		return new WP_Error( 'rest_forbidden', $error_msg, array( 'status' => rest_authorization_required_code() ) );
 	}
 
+	/**
+	 * Only administrators can access the API.
+	 *
+	 * @return bool|WP_Error True if a blog token was used to sign the request, WP_Error otherwise.
+	 */
+	public function require_author_privilege_callback() {
+		if ( current_user_can( 'publish_posts' ) ) {
+			return true;
+		}
+
+		$error_msg = esc_html__(
+			'You are not allowed to perform this action.',
+			'jetpack-publicize-pkg'
+		);
+
+		return new WP_Error( 'rest_forbidden', $error_msg, array( 'status' => rest_authorization_required_code() ) );
```

Comment:
We don't need to do all of this. WP REST API does it all for us, we can simply return `false` if the user doesn't have permission.

```suggestion
		return current_user_can( 'publish_posts' );
```

Same goes for `require_admin_privilege_callback` above.

Reply:
Thanks, I've updated the `require_author_privilege_callback` I've added to be like how you have recommended. 

I have left `require_admin_privilege_callback` the custom error messages we return for the rare use-case that someone might have built processing based on the error messages we return. Unlikely, but to maintain backwards compatibility, I haven't tweaked the error message we return there.

Reply:
On second thought, all routes but one use `require_author_privilege_callback`, so we are changing the error message we return, so I have gone ahead and updated it everywhere to a simple boolean return

Reply:
Thanks